src/components/Nav/Nav.Props.ts

```diff
@@ -36,6 +36,10 @@ export interface INavProps {
    * (Optional) The key of the nav item initially selected.
    */
   initialSelectedKey?: string;
+  /**
+   * (Optional) The nav container aria label.
+   */
+  navbarAriaLabel?: string;
```

Comment:
why not just call this ariaLabel?

Other components just use ariaLabel for the general container label.